packages/api/src/external/carequality/index.ts

```diff
@@ -4,6 +4,7 @@ const cqCommands = {
   patient: {
     discover: patient.discover,
     remove: patient.remove,
+    getCQData: patient.getCQData,
```

Comment:
nit: since we call cqCommands.[something] could this just be `getData` or even `get` since its implied its cq and patient

Reply:
`get: patient.getCQData`

Reply:
Went with `getData` but done. `patient.get` seems like it should return a patient. I even left a TODO in the `patient` field for CW as someone introduced a `get` function that *doesn't* return a patient...

Reply:
that TODO is done.